tests/batch/test_get_list_batches.py

```diff
@@ -0,0 +1,465 @@
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert result_header_signature in batch_ids
+
+
+def test_get_list_batches_with_invalid_ids():
+    """
+    Case: get a list of batches by invalid identifiers.
+    Expect: the following identifier is invalid error message.
+    """
+    invalid_id = '6f200'
+    batch_ids = '6f200995e766da7218ec2a3d0aeabbe1151128063cdf4e954cd08390a879b28e' \
+                '085a06f8708d2e6bb34f6501e8ddc981f0353627c1d4f90c80a656a8090c8751, ' \
+                f'{invalid_id}'
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'batch',
+        'get-list',
+        '--ids',
+        batch_ids,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    expected_error_message = {
+        'errors': {
+            'ids': [
+                f'The following identifier `{invalid_id}` is invalid.',
+            ],
+        },
+    }
+
+    assert FAILED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert dict_to_pretty_json(expected_error_message) in result.output
+
+
+def test_get_list_batches_with_start():
+    """
+    Case: get a list of batches by batch identifier starting from.
+    Expect: batches are returned starting from the batch with an identifier which matches specified start parameter.
+    """
+    start = 'fa2d1a209ad04fd2ad7fb5183976e647cc47b4c08e2e578097afc2566a0284e7' \
+            '60eb3f2ff8f72f290765211d4da3341f23091cc7a16805025a17c04a90818a44'
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'batch',
+        'get-list',
+        '--start',
+        start,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    result_header_signature = json.loads(result.output).get('result')[0].get('header_signature')
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert result_header_signature == start
+
+
+def test_get_list_batches_with_reverse():
+    """
+    Case: get a list of batches by reverse.
+    Expect: reversed list of batches is returned.
+    """
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'batch',
+        'get-list',
+        '--reverse',
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+
+
+def test_get_list_batches_by_head():
+    """
+    Case: get a list of batches by block identifier.
+    Expect: batches from specified block are returned.
+    """
+    head = '56100bf24eed12d2f72fe3c3ccf75fe2f53d87c224d9dda6fb98a1411070b06a' \
+           '40fcf97fccc61cb9c88442953af6ae50344ad7773f1becc6bae108443c18c551'
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'batch',
+        'get-list',
+        '--head',
+        head,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
```

Comment:
Add a check like this `assert result_header_signature == start`.

Reply:
This test is for head. Result doesn't include block identifier so there is nothing to compare with head.